pontoon/base/static/js/main.js

```diff
@@ -398,97 +269,43 @@ $(function () {
             }
         });
 
-    // Tabs
-    $('.tabs nav a').click(function (e) {
-        e.preventDefault();
-
-        var tab = $(this),
-            section = tab.attr('href').substr(1);
-
-        tab.parents('li')
-            .siblings()
-            .removeClass('active')
-            .end()
-            .addClass('active')
-            .end()
-
-            .parents('.tabs')
-            .find('section')
-            .hide()
-            .end()
-            .find('section.' + section)
-            .show();
-    });
-
-    // Toggle user profile attribute
-    $('.check-box').click(function () {
-        var self = $(this);
-
-        $.ajax({
-            url: '/api/v1/user/' + $('#server').data('username') + '/',
-            type: 'POST',
-            data: {
-                csrfmiddlewaretoken: $('#server').data('csrf'),
-                attribute: self.data('attribute'),
-                value: !self.is('.enabled'),
-            },
-            success: function () {
-                self.toggleClass('enabled');
-                var is_enabled = self.is('.enabled'),
-                    status = is_enabled ? 'enabled' : 'disabled';
-
-                Pontoon.endLoader(self.text() + ' ' + status + '.');
-
-                if (self.is('.force-suggestions') && Pontoon.user) {
-                    Pontoon.user.forceSuggestions = is_enabled;
-                    Pontoon.postMessage('UPDATE-ATTRIBUTE', {
```

Comment:
You dropped the postMessage here. I guess you know why, but should you also drop listening to `UPDATE-ATTRIBUTE` in `pontoon.js`, given this was the only send location?

Reply:
We've removed all other `postMessage` calls as part of removing Translate.Dead - most of them were in the `translate.js` file, which we removed completely.

We haven't changed pontoon.js as part of that process, so I suggest we keep it unchanged until we take on https://bugzilla.mozilla.org/show_bug.cgi?id=1536884.